docs/OpenAPIExamples.rst

```diff
@@ -0,0 +1,338 @@
+====================================
+Python API Examples (Open Endpoints)
+====================================
+
+The HCA API provides several ways for users of the Human Cell Atlas (HCA) to access and download
+data sets from the HCA. This page covers how to access the HCA using Python API bindings.
+
+*NOTE:* The HCA cli utility is compatible with Python 3.5+.
+
+
+create_version()
+----------------
+
+Returns a timestamp in ``DSS_VERSION`` format (e.g., ``1985-04-12T232050.520000Z``), useful for
+versioning bundles or files.
+
+.. note::
+  A version is a timestamp in RFC3339 format that keeps track of the most recent iteration of a
+  bundle or file. A bundle consists of many different data files, and both bundles and files have
+  version numbers.
+
+Example call to ``create_version()``:
+
+.. literalinclude:: ../test/tutorial/scripts/api/create_version_api.py
+
+
+download()
+----------
+
+Downloads a bundle to the local filesystem as a directory. By default, both data and metadata files
+are downloaded (flags can be added to download only the data or the metadata).
+
+Implementation detail: All files are downloaded to a local cache directory called ``.hca`` that is
+created in the directory where the download is initiated. The user should never need to interact
+directly with the ``.hca`` directory.
+
+See note above regarding version numbering.
+
+Example call to ``download()``:
+
+.. literalinclude:: ../test/tutorial/scripts/api/download_api.py
+
+Example response:
+::
+    {
```

Comment:
Adding newline above fixing parsing errors I see when viewing this doc on PyCharm. Also see main review comment.

Reply:
Thanks, fixed!